readme.md

```diff
@@ -168,72 +384,67 @@ var Hello = React.createClass({
   }
 });
 
-React.render(
+ReactDOM.render(
   <Hello name="Tony" age="21" />,
-  document.getElementById( "container" )
+  document.getElementById( "app" )
 )
 ```
 
-## Exercise: A Blog Post (15 / 45)
+> **NOTE:** The return statement in `render` can only return one DOM element. You can, however, place multiple elements within a parent DOM element, like we do in the previous example with `<div>`.
+
+## Exercise: A Blog Post
 
 Let's have some practice creating a React component for scratch. How about a blog post?
-* Define a Post constructor that has the following properties:
-  1. title
-  2. author
-  3. body
-  4. comments
+* Define a Post constructor in `index.js` that has the below properties. Then use it to generate an instance of a Post with the content of your choosing.
```

Comment:
Yeah, that's a good idea. I'll just have them hardcode a post. Now I gotta fix the commit history for the blog solution though... 😐